kubernetes/samples/scripts/domain-lifecycle/startServer.sh

```diff
@@ -0,0 +1,252 @@
+  withReplicas="INCREASED"
+  withPolicy="UNSET"
+  checkStartedServers "${domainJson}" "${serverName}" "${clusterName}" "${withReplicas}" "${withPolicy}" startsByReplicaIncreaseAndPolicyUnset
+  operation="INCREMENT"
+  createReplicaPatch "${domainJson}" "${clusterName}" "${operation}" incrementReplicaPatch replicaCount
+  if [[ -n ${managedServerPolicy} && ${startsByReplicaIncreaseAndPolicyUnset} == "true" ]]; then
+    # Server starts by increasing replicas and policy unset, increment and unset
+    printInfo "Unsetting the current start policy '${managedServerPolicy}' for '${serverName}' and incrementing replica count."
+    createPatchJsonToUnsetPolicyAndUpdateReplica "${domainJson}" "${serverName}" "${incrementReplicaPatch}" patchJson
+    action="PATCH_REPLICA_AND_UNSET_POLICY"
+  elif [[ -z ${managedServerPolicy} && ${startsByReplicaIncreaseAndPolicyUnset} == "true" ]]; then
+    # Start policy is not set, server starts by increasing replicas based on effective policy, increment replicas
+    printInfo "Updating replica count for cluster '${clusterName}' to ${replicaCount}."
+    createPatchJsonToUpdateReplica "${incrementReplicaPatch}" patchJson
+    action="PATCH_REPLICA"
+  else
+    # Patch server policy to always and increment replicas
+    printInfo "Patching start policy of server '${serverName}' from '${effectivePolicy}' to 'ALWAYS' and \
+incrementing replica count for cluster '${clusterName}'."
+    createPatchJsonToUpdateReplicaAndPolicy "${incrementReplicaPatch}" "${alwaysStartPolicyPatch}" patchJson
+    action="PATCH_REPLICA_AND_POLICY"
+  fi
+elif [[ -n ${clusterName} && "${keepReplicaConstant}" == 'true' ]]; then
+  # Replica count needs to stay constant, check if server starts by unsetting policy
+  withReplicas="CONSTANT"
+  withPolicy="UNSET"
+  checkStartedServers "${domainJson}" "${serverName}" "${clusterName}" "${withReplicas}" "${withPolicy}" startsByPolicyUnset
+  if [[ "${effectivePolicy}" == "NEVER" && ${startsByPolicyUnset} == "true" ]]; then
+    # Server starts by unsetting policy, unset policy
+    printInfo "Unsetting the current start policy '${effectivePolicy}' for '${serverName}'."
+    createPatchJsonToUnsetPolicy "${domainJson}" "${serverName}" patchJson
+    action="UNSET_POLICY"
+  else
+    # Patch server policy to always
+    printInfo "Patching start policy for '${serverName}' to '${targetPolicy}'."
+    createPatchJsonToUpdatePolicy "${alwaysStartPolicyPatch}" patchJson
+    action="PATCH_POLICY"
+  fi
+else
+  # Server is an independent managed server
+  printInfo "Unsetting the current start policy '${effectivePolicy}' for '${serverName}'."
+  createPatchJsonToUnsetPolicy "${domainJson}" "${serverName}" patchJson
+  action="UNSET_POLICY"
+fi
+
+if [ "${verboseMode}" == "true" ]; then
+  printInfo "Patching domain with Json string -> ${patchJson}"
+fi
+${kubernetesCli} patch domain ${domainUid} -n ${domainNamespace} --type='merge' --patch "${patchJson}" 
+
+case ${action} in
```

Comment:
Now that the script is simplified, it comes to mind that the following printInfos are redundant given that all paths to this point already have a printInfo that says what the script is about to do before calling patch.  If you agree, consider replacing the entire 'case' statement with 'Patch command succeeded' or similar... (Ditto for the stopServer.sh code).